commands/cmd-shell/src/lib.rs

```diff
@@ -0,0 +1,63 @@
+use anyhow::{Context, Result};
+use async_trait::async_trait;
+use clap::{ArgMatches, Clap};
+use oro_command::OroCommand;
+use oro_command::OroCommandLayerConfig;
+use oro_config::OroConfig;
+use oro_error_code::OroErrCode as Code;
+use std::path::PathBuf;
+use std::str::FromStr;
+
+#[derive(Debug, Clap)]
+pub struct ShellCmd {
+    #[clap(long, default_value = "node")]
+    node: String,
+
+    #[clap(long)]
+    data_dir: Option<PathBuf>,
+
+    #[clap(multiple = true)]
+    args: Vec<String>,
+
+    #[clap(from_global)]
+    loglevel: log::LevelFilter,
+    #[clap(from_global)]
+    json: bool,
+    #[clap(from_global)]
+    quiet: bool,
+}
+
+#[async_trait]
+impl OroCommand for ShellCmd {
+    async fn execute(self) -> Result<()> {
+        dbg!(&self);
+        Ok(())
+    }
+}
+
+impl OroCommandLayerConfig for ShellCmd {
```

Comment:
You don't need this. Just add `#[derive(OroCommand)]` to `struct ShellCmd` and you should be good on this front :)